test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -1191,7 +1191,7 @@ def forward(self, x):
             match_nodes = 12
             self._test_common(mod, (v,), match_count, match_nodes, rtol=1e-2, atol=1e-2)
 
-    def test_qconv2d_maxpool2d_linear_dynamic(self):
+    def test_qconv2d_maxpool2d_linear_dynamic_cpu(self, include_ops=None):
```

Comment:
The included ops are intended to be hard-coded for the tests. Any reason why we want to make them customizable?

Reply:
The included ops strings in the generated code are different for python wrapper and cpp wrapper so that I made it customizable to pass in different strings for python and cpp wrapper test.

For example, for python wrapper, the generated code will be like below and the included_ops strings will be `torch.ops.onednn.qconv2d_pointwise`.
```
buf1 = torch.ops.onednn.qconv2d_pointwise(buf0, 0.024776775389909744, 97, _frozen_param10, _frozen_param4, _frozen_param5, _frozen_param1, [1, 1], [1, 1], [1, 1], 1, 95.88209060714476, 0, False, 'relu', [], '')
```

While when using cpp wrapper, the generated code will be like the following and if we still check `torch.ops.onednn.qconv2d_pointwise`, the test will fail. We need to provide a different string here:
```
static auto op_qconv2d_pointwise = c10::Dispatcher::singleton()
	.findSchemaOrThrow("onednn::qconv2d_pointwise", "")
	.typed<
        at::Tensor(
            at::Tensor act,
            double act_scale,
            int64_t act_zero_point,
            at::Tensor weight,
            at::Tensor weight_scales,
            at::Tensor weight_zero_points,
            c10::optional<at::Tensor> bias,
            torch::List<int64_t> stride,
            torch::List<int64_t> padding,
            torch::List<int64_t> dilation,
            int64_t groups,
            double inv_output_scale,
            int64_t output_zero_point,
            bool fp32_output,
            c10::string_view attr,
            torch::List<c10::optional<at::Scalar>> scalars,
            c10::optional<c10::string_view> algorithm)>();
auto buf1 = op_qconv2d_pointwise.call(buf0, 0.024776775389909744, 97L, _frozen_param10, _frozen_param4, _frozen_param5, _frozen_param1, {1L, 1L}, {1L, 1L}, {1L, 1L}, 1L, 95.88209060714476, 0L, false, "relu", {c10::nullopt}, "");
```